### PR TITLE
Cleanup backups

### PR DESCRIPTION
**Pull request by jonstacks**
Set a retention policy for the number of snapshots and images to keep.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,5 +63,29 @@ make create-lambda-ebs-backup
...
+lambda function that we are to create a snapshot of the volume.
+
+
+#### Controlling how many Volume Snapshots are kept
+
+By default, only 2 volume snapshots are kept. You can override this globally
+by setting the environment variable `DEFAULT_MAX_KEEP_SNAPSHOTS` in the
+cloudformation for the lambda function. Alternatively, you can control this on
+a volume by volume basis by setting the following tag key:value pair on the
+volume: `lambda-ebs-backup/max-keep-snapshots`.
```
Comment (bksteiny) [inline README.md:76]: If both are defined, I assume the tag takes precedence over the environment variable?https://github.com/C2FO/lambda-ebs-backup/pull/2/files#diff-1de92dee00ed13c236d39c6583aa8329R64

File: README.md [hunk 2]
```diff
@@ -63,5 +63,29 @@ make create-lambda-ebs-backup
 ### Tagging Volumes for backup
 
 In order to backup a volume, you must opt-in by setting a tag key:value pair on
-the volume. By default, this is `lambda-ebs-backup:true`. This tells the lambda
-function that we are to create a snapshot of the volume.
+the volume. By default, this is `lambda-ebs-backup/backup:true`. This tells the
+lambda function that we are to create a snapshot of the volume.
+
```
Comment (bksteiny) [inline README.md:68]: Might be useful to document the tag for reboots here as well: `lambda-ebs-backup/reboot-on-image`

**Review by bksteiny — APPROVED** [review]
💯

File: README.md (hunk 1 above)
Comment (jonstacks) [reply README.md:76]: @bksteiny, that is correct. The global default is 2 unless overwritten by the environment variable. You can further change this behavior on a volume by volume basis by the value of that tag to whatever you want.

File: README.md (hunk 2 above)
Comment (jonstacks) [reply README.md:68]: Will do.

**Review by tbplayer7 — APPROVED** [review]
This is awesome.

**Pull request merged**